src/Symfony/Component/Validator/Tests/Constraints/CardSchemeValidatorTest.php

```diff
@@ -102,6 +102,12 @@ public function getValidNumbers()
             array('MAESTRO', '6594371785970435599'),
             array('MASTERCARD', '5555555555554444'),
             array('MASTERCARD', '5105105105105100'),
+            array('MASTERCARD', '2221005555554444'),
+            array('MASTERCARD', '2230000000000000'),
+            array('MASTERCARD', '2300000000000000'),
```

Comment:
If we are testing `23xxx`, `26xxx` and `27xxx` maybe we should also test the missing `24xxx` and `25xxx` to avoid future regressions (these regexps are a bit tricky).